acceptance/config/nodes/win2012r2-rubyx64.yaml

```diff
@@ -12,6 +20,12 @@ HOSTS:
     ruby_arch: x64
     hypervisor: vcloud
     template: win-2012r2-x86_64
+    puppetconfdir: C:/ProgramData/PuppetLabs/puppet/etc
+    puppetcodedir: C:/ProgramData/PuppetLabs/puppet/code
+    puppetvardir: C:/ProgramData/PuppetLabs/puppet/var
+    distmoduledir: C:/ProgramData/PuppetLabs/puppet/code/modules
+    sitemoduledir: C:/opt/puppetlabs/puppet/modules
+    hieraconf: C:/ProgramData/PuppetLabs/puppet/etc/hiera.yaml
```

Comment:
I don't think this file is actually used on the windows agents (perhaps during `puppet apply`), but for consistency this should be `C:/ProgramData/PuppetLabs/puppet/code/hiera.yaml`